### PR TITLE
[Automated] Update actions

### PR DESCRIPTION
**Pull request by mattmoor**
Produced via:
```shell
# meta: knative/.github
# main: knative/client
cp $(find "${GITHUB_WORKSPACE}/meta/workflow-templates" -type f -name '*.yaml') \
  "${GITHUB_WORKSPACE}/main/.github/workflows/"
```
/cc rhuss maximilien
/assign rhuss maximilien

### CODE REVIEW

**Comment by rhuss** [comment]
@mattmoor what would we need to do to activate the workflow actions ? they don't show up in this PR.

In general, it looks good to me, especially if we can fine tune the rules of those checks. 👍 from my side.

**Review by maximilien — APPROVED** [review]
LGTM

/retest

File: .github/workflows/knative-boilerplate.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        - extension: sh
+          language: Bash
+
+    steps:
+
+      - name: Set up Go 1.15.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15.x
+        id: go
```
Comment (navidshaikh) [inline .github/workflows/knative-boilerplate.yaml:49]: is this required for following review tools? or bumping to go 1.15 for go.mod is in order ?

Comment (mattmoor) [reply .github/workflows/knative-boilerplate.yaml:49]: It's used to `go get`.

**Comment by mattmoor** [comment]
I'm going to close this, since the new naming scheme PRs should replace these.

**Pull request closed without merging**